<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iCubesClass</class>
 <widget class="QMainWindow" name="iCubesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelModule">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Module</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboboxModule">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>BinaryMath</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ColorPalette</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="buttonConfigure">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>761</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Canvas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartstopButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
